CHANGELOG.rst

```diff
@@ -16,6 +16,14 @@ Unreleased
 
 *
 
+[5.2.0]
+~~~~~~~
+
+* Added Support for Django4 and used djfernet instead of fjango-fernet-fields
```

Comment:
is this is a typo? `fjango-fernet-fields`?

Reply:
yes it is. django-fernet-fields was intended. I will correct that